jenkins/scripts/windows/node-compile-windows.cmd

```diff
@@ -15,7 +15,7 @@ call %~dp0compile.cmd
 if errorlevel 1 exit /b
 
 :: Select files to pack
-set "BINARY_FILES=config.gypi icu_config.gypi Release/node.exe Release/node.lib Release/openssl-cli.exe Release/cctest.exe Release/node.pdb"
+set "BINARY_FILES=config.gypi icu_config.gypi Release/node.exe Release/node.lib Release/openssl-cli.exe Release/cctest.exe Release/embedtest.exe Release/node.pdb"
```

Comment:
I'd recommend putting `embedtest.exe` behind an `if exist` check like the lines below otherwise you'll have to wait for the PR that adds `embedtest.exe` to be backported to all supported release lines before this change could land.

Reply:
I can do it, but the `embedtest.exe` is already being built. It is part of an existing test,
I am just trying to enable it running on Windows.

Reply:
I have updated the script to make use of `embedtest.exe` conditional.